src/common/date.ts

```diff
@@ -0,0 +1,16 @@
+import { isEqual } from 'date-fns';
+
+export const isDateOnlyEqual = (left: Date, right: Date): boolean => {
```

Comment:
There's a a `date-fns` function called `isSameDay` I believe this is what you're looking for

Reply:
Oh, nice. Yes, I thought they didn't have this common requirement.